app/assets/javascripts/discourse/app/routes/wizard.js

```diff
@@ -1,31 +1,31 @@
 import Route from "@ember/routing/route";
```

Comment:
Note that this file lives in the normal location (`app/routes/wizard.js`). With `splitAtRoute: ["wizard"]`, `app/routes/wizard`, `app/routes/wizard/*`, `app/controllers/wizard`, `app/controllers/wizard/*`, `app/templates/wizard`, `app/templates/wizard/*` all get split off into the lazy bundle. In an earlier iteration, I experimented with putting all of these files primarily in `app/static/wizard`, and added a stub to re-export them in the normal location, but that ended up being a lot of extra boilerplate IMO.